### ISSUE TITLE
VN replace huron support (ammo, fuel, repair) containers by slingloadable versions

### ISSUE THREAD

**Issue by SunStalker** [SunStalker/KP-Liberation#11]
missionframework\presets\blufor\vn.sqf
support_vehicles = [
    ["B_Slingload_01_Repair_F",275,0,0],                                // Huron Repair
    ["B_Slingload_01_Fuel_F",75,0,200],                                 // Huron Fuel
    ["B_Slingload_01_Ammo_F",75,200,0]                                  // Huron Ammo
];

currently the vn repairstations are empty pallets and not slingloadable.

short search found no better candidates yet

**Comment by FatRefrigerator** [comment]
This should already be the case, all of those are already in the build list.

**Comment by SunStalker** [comment]
yes, the huron ones are in the build list, BUT they are not fitting the setting, so this task is just a reminder to replace them as soon as better fitting ons are available. 

I will also test the behaviour with the objects which are slingloadable, eg the standard ammo boxes etc
